src/main/java/com/microsoft/graph/serializer/GsonFactory.java

```diff
@@ -69,6 +69,17 @@ private GsonFactory() {
      * @return the new instance
      */
     public static Gson getGsonInstance(final ILogger logger) {
+        return getGsonInstance(false, logger);
+    }
+
+    /**
+     * Creates an instance of GSON
+     *
+     * @param serializeNulls the setting of whether or not to serialize the null values in the JSON object
+     * @param logger         the logger
+     * @return the new instance
+     */
+    public static Gson getGsonInstance(final boolean serializeNulls, final ILogger logger) {
```

Comment:
same parameters ordering question

Reply:
also, while you're here, would you please add:
- nonnull annotations for the logger parameters
- ObjectRequireNonNull for the logger parameters
- Nonnull annotations for the return value